addon/components/o-s-s/phone-number-input.ts

```diff
@@ -10,6 +11,8 @@ interface OSSPhoneNumberInputArgs {
   prefix: string;
   number: string;
   placeholder?: string;
+  errorMessage?: string;
+  hasError?: boolean;
```

Comment:
In which case can you have an error on the input without an error message?

Reply:
Sometimes during form validation we want to display only a red border ; This parameter is also defined for OSS::InputContainer 😇